Detailed notes ENCODE Cancer 09/22/2014
===

>By Jason Bedford

###About applying for DB Gap access with ENCODE GWAS

* Having a joint application would make things easier

* We could not share any variants found in the data sets. We could talk about a single SNP but not a single patent.

* What data sets would we both be interested in?

* Will discuss on the call 10/13

###Leonard Lipovich RNA in Breast cancer: Introduction

***Slide 04***

* Long non-coding RNAs are not new. But mRNAs have been getting all the attention

***Slide 05***

* Despite knowing about long non-coding RNAs for some time there are only 100 that have been deeply studied. 

* Present in almost all compartment of the cell.

* LncRNA diverse and heterogeneous functions including: endogenous RNA regulators of transcription factors, enhancers as lncRNA TSS.

* As RNAs are studies more we are learning how complex they really are. 

* See Kung et al 2013, Genetics 193: 651-669 for a nice review. 

***Slide 06***

* Non-coding RNA genes are more numerous than protein-coding genes in the human genome.

***Slide 07***

* The connection between lncRNA and disease has been well established in many papers. 

***Slide 08***

* LncRNA genes and cancer

* Highly Unregulated in Liver Cancer: a “poster child” and a “typical cancer lncRNA”!

* RNAi of HULC has reproducible effects including up regulation of CDK8 and a tumor suppressor candidate

***Slide 09***

* The RNA in #2 is MALAT-1 (NEAT-2), a 7-kb lncRNA: a predictor of lung cancer metastasis, of survival in osteosarcoma, associated with SC35 splicing domains, an essential polyadenylated component of nuclear speckles

***Slide 10***

* LncRNA Comics has emerged as a clinically relevant field

* DOXORUBICIN SENSITIVITY, Joerger, A.C. & Fersht, A.R. Oncogene 26, 2226–2242 


***Slide 11***

* 606 papers in just the first 9 months of 2014

***Slide 12***

* LL uses a custom microarrays

***Slide 13***

* ￼￼our custom human lncRNA microarray

* 7 60mer probes per gene

* Labeled antisense cRNA

* Two-color; Agilent 4x44k; dye-flip quadruplicate format

***Slide 14***

* Description microarray: two-color dye-flip quadruplicate custom lncRNA microarray experiment

***Slide 15***

* Advantages of LL custom microarray: the foundation for our human lncRNA custom microarray – is highly non redundant relative to the Broad Institute lincRNA and
The ENCODE Project lncRNA datasets

***Slide 16***

* The apoptosis-proliferation axis: regulation by Estrogen Receptor α (ERα), a nuclear hormone receptor

* E2-induced ERα direct targets: putative oncogenes 

* E2-repressed ERα direct targets: putative tumor suppressors

***Slide 17***

* The human estrogen-responsive lncRNA Home

* 5,586 lncRNAs that we characterized (Jia et al 2010, RNA 16:1478-87) are represented on our custom human lncRNA microarray (Lipovich et al 2012, Genetics 192: 1133-1148)

* MCF7 human ERα+ breast cancer cell line: treated with 10 nM E2(17-β-estradiol)for 24 h (C.Y.Lin,University of Houston,TX) and compared with untreated controls
(exactly as in: Lin et al 2007, PLoS Genetics 3[6]:e87)

* 127 differentially expressed lncRNAs: >1.4 greater fold change, <5% FDR, all 7 unique probes (per lincRNA gene) are differentially expressed in the same direction

***Slide 18***

* list of E2 induced and repressed lncRNAs from the custom microarray. 

###RNAs in breast cancer: Estrogen-induced lncRNAs

***Slide 20***

* Estrogen-induced lncRNAs: CR 593775 is the Number 1 hit from our custom microarray. List of 127 E2-responsive DE lncRNAs from MCF7 cells.

***Slide 21***

* Estrogen-induced human lncRNAs that affect the growth of MCF 7 breast cancer cells have primate-specific sequences and young gene structures: CR 593775, our top hit

* ENCODE ERα binding site: confirmation that CR 593775 is not only an estrogen-induced lncRNA, but also a direct ERα target at the TS


***Slide 22***

* Estrogen-induced human lncRNAs that affect the growth of MCF 7 breast cancer cells have primate-specific sequences and young gene structures: CR 593775, our top hit

* Splice Acceptor : only in primates; arose after the prosimian split, Splice Donor: tree shrew + primates

***Slide 23***

* SiRNA-based reverse genetics of E2-induced human lncRNAs

* cell counts decrease, consistent with cell death, effect is consistent across most of the top 16 maximally induced lncRNAs


***Slide 24***

* Knockdown of the lncRNA CR 593775 with two different siRNAs reduces BrdU incorporation in MCF7 cells, we see that there is a phenotype in the knockdown. 

***Slide 25***

* ￼SiRNA knockdown of estrogen-induced lncRNAs: crystal violet assay, whole-well pictures
(120 h post-transfection, 96 h post-E2). See image on slide 25

###Leonard Lipovich: Estrogen-repressed lncRNAs
￼

***Slide 27***

* Overexpression of estrogen-repressed lncRNAs in T47D cells reveals reduced cell growth with and without E2

***Slide 28***

* Overexpression of estrogen-repressed lncRNAs in T47D cells reveals reduced cell growth with and without E2 (crystal violet and MTT assays)


***Slide 29***

* BC041455 is our top estrogen-repressed human lncRNA

***Slide 30***

* The transcription factor CEBPβ is a known inducer of cell death in breast cancer cells. (Abreu and Sealy 2010, Exp Cell Res 316:3227-38

* Does the promoter of BC041455 bind this transcription factor in ENCODE cell lines?

***Slide 31***

* CEBB ChIP-seq peak is localized to the BC041455 TSS in ENCODE data

***Slide 32***

* Direct regulation of E2-responsive lncRNA genes by CEBPβ binding is a global trend 

***Slide 33/34***

Both splice sites of BC041455’s sole intron are primate-specific

***Slide 35/36***

* BC041455, a primate-specific lncRNA, may impact the MAP kinase pathway

* the lncRNA reduces Erk1 and Erk2 phosphorylation, consistent with the observed reduction in cell proliferation and viability

* a primate-specific lncRNA providing a regulatory input into a central cancer pathway?

***Slide 37***

* evolution summary of the E2-responsive human lncRNA Home

***Slide 38***

* The lncRNA perspective on “animal models of human disease: Is human breast cancer,
to some extent, a primate-specific disease?

### Conclusions

* LncRNA aware “theranostics” for post-genomic personalized medicine

* More than 90% of GWAS significant variants are non-coding,
including some in lncRNA exons and many in lncRNA introns, lncRNA promoter

* lncRNAs remain an underappreciated cause of disease

* non-conserved (lncRNA) therapeutic targets = fewer off target effects? (lncRNAs are distant outlier nodes, connected by long edges to conserved regulatory network cores)

* genome editing and/or post-transcriptional targeting of lncRNAs (and of their interaction with other molecules) = new treatments and cures?

*  lncRNAs are master regulators in gene networks


###Questions 

***Ekta to LL***: the GEncode annotations are biased to spliced genes. 

* LL completely agrees says that we should use this annotations addition to others. 

***Mark to LL***: the genes are not primate specific but only the splice sites 

* yes the gene structure is primate specific. 

* What LL does with a transcript is map back to the genome then do genome comparison across different species. 

***Mark to LL***: How is it that you call these lncRNAs master regulators when they don't really have all that many direct regulatory targets. 

* the idea is that lncRNAs regulate genes, which are central to the network.

***Mark to LL***: but some of those gene also regulate the original lncRNA to we have a chicken and egg problem. 

* yes we have seen exactly this experimentally. 

* and yes it is difficult to know which comes first. 

### Final Thoughts

* LL would be happy to collaborate with anyone in the consortium.

* next call will be 10/13 well send out notes and a reminder. 



